Module-generation/loadModules.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Supporting script
+
+MODULEDIR="./module-site"
+MODULES="./modules"
+mkdir $MODULEDIR
+cd $MODULEDIR
+git clone https://github.com/MovingBlocks/ModuleSite.git
+cd ModuleSite
+rm -R modules
+cd ..
+cd ..
+cp -r $MODULES ./module-site/ModuleSite
+cd ./module-site/ModuleSite
+git config --global user.email "example@gmail.com"
+git config --global user.name "userName"
+git checkout -b module_gen
+git add .
+git commit -m "add: all modules "
+git push https://${GIT_CREDS}@github.com/MovingBlocks/ModuleSite.git module_gen -f
```

Comment:
this seems less like "loading" modules, but rather like "uploading" them :thinking:

Reply:
it will clone ModuleSite and discard the old modules and load new modules and create PR to ModuleSite